packages/appstore/src/modules/tour-guide/tour-guide.tsx

```diff
@@ -31,7 +30,7 @@ const TourGuide = () => {
     const history = useHistory();
     const [joyride_index, setJoyrideIndex] = React.useState<number>(0);
     const tour_step_locale = getTourStepLocale();
-    const high_risk_tour_step_locale = getHighRiskTourStepLocale();
+    const high_risk_tour_step_locale = getTourStepLocale();
```

Comment:
Based on ur investigation above, u can make or keep the change

Reply:
change can be made 👍🏼